assets/scss/templates/_reportes.scss

```
@@ -1,13 +1,14 @@
 
 .dp-wrapper-messages {
-  max-width: $dp-wrapper-max-width;
+  max-width: 960px; /* check to modularize the messages */
```

Please change it in order to search it easily:

```suggestion
  max-width: 960px; /* TODO: check to modularize the messages */
```